### ISSUE TITLE
Visually Hidden Headings Should Have Titles Exposed on Focus/Hover

### ISSUE THREAD

**Issue by mgifford** [microsoft/accessibility-insights-web#585]
## Describe the feature

There are visually hidden headings which should have the semantic information exposed on focus & hover (when highlighting the H2) in the example below. 

Screen reader content should be visible.

**To Reproduce**

1. Go to https://tastybackend.com/
2. Highlight the headings.

## Expected behavior

Display semantic content if accessible to screen readers.

## Screenshots

<img width="1404" alt="Screen Shot 2019-04-22 at 3 27 41 PM" src="https://user-images.githubusercontent.com/116832/56538639-99d22600-6531-11e9-948b-56bbfb493f53.png">
<img width="1420" alt="Screen Shot 2019-04-22 at 3 26 13 PM" src="https://user-images.githubusercontent.com/116832/56538641-99d22600-6531-11e9-9d87-2739dd7a9325.png">

## Extension (please complete the following information)

- OS: MacOSX
- Chrome Version: Latest
- Target Page: https://tastybackend.com/

**Comment by haonliu** [comment]
@mgifford Thanks a lot for the clear description of this feature request! This is a great suggestion, we will look into this feature. Before the feature is available, as an alternative, you can use the headings Assessment test to visualize the semantic information of visually hidden headings: 
![heading](https://user-images.githubusercontent.com/15974344/56539251-2a047100-651a-11e9-86b6-8a71ff2c7b54.gif)

Hope that helps!

**Comment by mgifford** [comment]
Nice..  That's a great feature too!
